packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js

```diff
@@ -46,7 +47,7 @@ const documentTypeMap = {
 		APPEAL_DOCUMENT_TYPE.ORIGINAL_APPLICATION_FORM,
 	[documentTypes.whoWasNotified.name]: APPEAL_DOCUMENT_TYPE.WHO_NOTIFIED,
 	[documentTypes.uploadSiteNotice.name]: APPEAL_DOCUMENT_TYPE.WHO_NOTIFIED_SITE_NOTICE,
-	[documentTypes.uploadLettersToNeighbours.name]:
+	[documentTypes.uploadNeighbourLetterAddresses.name]:
```

Comment:
duplicate doc type + question, have removed all references to the duplicate to avoid future confusion